dist/pythonlibs/testrunner/__init__.py

```diff
@@ -7,51 +8,17 @@
 # directory for more details.
 
 import os
-import signal
 import sys
-import subprocess
-import time
-from traceback import extract_tb, print_tb
-
+from traceback import print_tb
 import pexpect
 
-PEXPECT_PATH = os.path.dirname(pexpect.__file__)
-RIOTBASE = (os.environ.get('RIOTBASE') or
-            os.path.abspath(os.path.join(os.path.dirname(__file__),
-                                         "..", "..", "..")))
-
-# Setting an empty 'TESTRUNNER_START_DELAY' environment variable use the
-# default value (3)
-MAKE_TERM_STARTED_DELAY = int(os.environ.get('TESTRUNNER_START_DELAY') or 3)
-
-
-def list_until(l, cond):
-    return l[:([i for i, e in enumerate(l) if cond(e)][0])]
-
-
-def find_exc_origin(exc_info):
-    pos = list_until(extract_tb(exc_info),
-                     lambda frame: frame[0].startswith(PEXPECT_PATH)
-                     )[-1]
-    return (pos[3], os.path.relpath(os.path.abspath(pos[0]), RIOTBASE), pos[1])
+from .spawn import find_exc_origin, setup_child, teardown_child
+from .unittest import PexpectTestCase
```

Comment:
This is unused ?

Reply:
Yes, but this is also the top-level of the module. So people importing `testrunner` don't have to import `testrunner.PexpectTestCase` additionally.

Reply:
I usually `import package.module` in my code anyway, but I do not mind having this here.

Reply:
Ok, I see.

Reply:
Told `flake8` to ignore it now also.